cli_reference/get_started_cli.adoc

```diff
@@ -153,27 +203,12 @@ service frontend-test (172.30.17.236:5432 -> 8080)
     #1 deployment waiting on image
 
 To see more information about a service or deployment config, use 'osc describe service <name>' or 'osc describe dc <name>'.
-You can use 'osc get pods,svc,dc,bc,builds' to see lists of each of the types described above. 
+You can use 'osc get pods,svc,dc,bc,builds' to see lists of each of the types described above.
 ----
+====
+
+== What's Next?
 
-== CLI Object Types
-OpenShift supports the following object types, some of which have abbreviated syntax that you can use with `osc`.
-
-.Supported Object Types
-[options="header"]
-|===
-
-|Object Type |Abbreviated Version
-
-|build |
-|buildConfig | `bc`
-|deploymentConfig | `dc`
-|imageStream | `is`
-|imageStreamTag | `istag`
-|imageStreamImage | `isimage`
-|event |`ev`
-|minion |`mi`
-|pod |`po`
-|replicationController |`rc`
-|service |`svc`
-|===
\ No newline at end of file
+After you have link:get_started_cli.html#basic-setup-and-login[logged in], check
+out some link:basic_cli_operations.html[basic CLI operations], or get straight
+into link:../dev_guide/new_app.html[creating a new application].
```

Comment:
This is not appropriate wording for Enterprise docs:

"Check out some....., or get straight into....."

Please modify this.